### ISSUE TITLE
Version 2.5.3 breaks windows test runs

### ISSUE THREAD

**Issue by hmn** [test-kitchen/test-kitchen#1665]
## :ghost: Brief Description

After updating to chef workstation 20.7.96 (it includes test-kitchen 2.5.3) I can't run my test kitchen tests. If I downgrade to 2.5.2 it works.

## Version

2.5.3

## Scenario

I get the error "A parameter cannot be found that matches parameter name 'tmp_dir'." and it looks like these params where added in 2.5.3

- https://my.diffend.io/gems/test-kitchen/2.5.2/2.5.3
- https://github.com/chef/chef-workstation/pull/1337/files
- https://github.com/test-kitchen/test-kitchen/commit/c63819fca968140cf9fa42e5369f1283c0b7f0ed

It looks like the error occurs because tmp_dir is not a param to the function Install-Project in https://omnitruck.chef.io/install.ps1

## Steps to Reproduce

Run kitchen test on a windows host (I tested on windows 2019 with the tas50/windows_2019 box)

My kitchen.yml looks like this
```
---
driver:
  name: vagrant

provisioner:
  name: chef_zero
  product_version: 16.1.16
  product_name: chef
  install_strategy: always
  always_update_cookbooks: true

verifier:
  name: inspec

platforms:
  - name: windows-2019
    driver:
      box: tas50/windows_2019
      customize:
        memory: 1536

suites:
  - name: default
    data_bags_path: "test/integration/data_bags"
```

## Error

```
       Preparing client.rb
       
       ModuleType Version    Name                                ExportedCommands                                                                                                                                                                               
       ---------- -------    ----                                ----------------                                                                                                                                                                               
       Script     0.0        Omnitruck                           {Get-ProjectMetadata, Install-Project, install}                                                                                                                                                
$$$$$$ Exception setting "OutputEncoding": "The handle is invalid.
$$$$$$ "
At line:2 char:1
$$$$$$ + [Console]::OutputEncoding = New-Object -typename System.Text.ASCIIEnc ...
$$$$$$ + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting
A parameter cannot be found that matches parameter name 'tmp_dir'.
At line:479 char:83
$$$$$$ + ... n 16.1.16 -channel stable -install_strategy 'always' -tmp_dir '$env:T ...
$$$$$$ +                                                          ~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Install-Project], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Install-Project

>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Converge failed on instance <default-windows-2019>.  Please see .kitchen/logs/default-windows-2019.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

**Comment by gsreynolds** [comment]
I believe this is fixed in #1664 and will presumably make it's way into a 2.5.4 release

**Comment by tyler-ball** [comment]
Closing this because we reverted the change and released it in TK 2.5.4. That will go out in the next Workstation release early next week. We'll take another stab at fixing the original issue then too!
